### PR TITLE
Complete iteration zero

### PR DESCRIPTION
**Pull request by orlov-n**
Please review all the functions and variables.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -14,6 +16,15 @@ var currentCover;
 
 
 // We've provided one function to get you started
+randomizeCover()
```
Comment (laytonmaes) [inline src/main.js:19]: randomizes cover for use later

File: src/main.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 // Create variables targetting the relevant DOM elements here 👇
-
-
+var coverImage = document.querySelector('.cover-image')
```
Comment (laytonmaes) [inline src/main.js:2]: variables that pull from dom

File: styles.css [hunk 3]
```diff
@@ -43,13 +43,16 @@ body {
   margin: 4vh auto;
   position: relative;
   width: 26vw;
+  border: red solid 1px;
```
Comment (laytonmaes) [inline styles.css:46]: we need to remember to delete all the red

Comment (laytonmaes) [reply styles.css:46]: borders

File: src/main.js (hunk 1 above)
Comment (orlov-n) [reply src/main.js:19]: Do you even lift bro?

**Comment by laytonmaes** [comment]
everything looks good, we want to remember to get rid of red border.

**Pull request merged**